ui/imports/shared/status/StatusChatInput.qml

```diff
@@ -461,11 +458,37 @@ Rectangle {
                 const clipboardText = Utils.plainText(QClipboardProxy.text)
                 const copiedText = Utils.plainText(d.copiedTextPlain)
                 if (copiedText === clipboardText) {
-                    d.internalPaste = true
+                    if (d.copiedTextPlain.includes("@")) {
+                        d.copiedTextFormatted = d.copiedTextFormatted.replace(/span style="/g, "span style=\" text-decoration:none;")
+
+                        let lastFoundIndex = -1
+                        for (let j = 0; j < d.copiedMentionsPos.length; j++) {
+                            const name = d.copiedMentionsPos[j].name
+                            const indexOfName = d.copiedTextPlain.indexOf(name, lastFoundIndex)
+                            lastFoundIndex += name.length
+
+                            if (indexOfName === d.copiedMentionsPos[j].leftIndex + 1) {
+                                const mention = {
+                                    name: name,
+                                    pubKey: d.copiedMentionsPos[j].pubKey,
+                                    leftIndex: (d.copiedMentionsPos[j].leftIndex + d.copyTextStart - 1),
+                                    rightIndex: (d.copiedMentionsPos[j].leftIndex + d.copyTextStart + name.length)
+                                }
+                                mentionsPos.push(mention)
+                                d.sortMentions()
+                            }
+                        }
+                    }
+
+                    const prevLength = messageInputField.length
+                    insertInTextInput(d.copyTextStart, d.copiedTextFormatted)
+                    messageInputField.cursorPosition = d.copyTextStart + messageInputField.length - prevLength
                 } else {
                     d.copiedTextPlain = ""
                     d.copiedTextFormatted = ""
                     d.copiedMentionsPos = []
+                    insertInTextInput(d.copyTextStart, QClipboardProxy.text)
```

Comment:
This will effectively revert my fixes done in here: https://github.com/status-im/status-desktop/pull/8945

I don't get it... why isn't this creating a git conflict? :/

Reply:
you are right, brought back